_content/essays/opening-door-of-dharma_chokyi-lodro.md

```diff
@@ -15,8 +16,11 @@ pages: 10
 ---
 
 > There are thus a great many systems of Dharma teaching in Tibet,
+
 But aside from their nominal variations,
-There is really no significant difference between them—
+
+There is really no significant difference between them
```

Comment:
Two _spaces_ not new lines. We went over this during our last call...

https://kramdown.gettalong.org/syntax.html#paragraphs